drivers/si70xx/si70xx.c

```diff
@@ -219,3 +225,18 @@ void si70xx_get_both(const si70xx_t *dev, uint16_t *humidity, int16_t *temperatu
 
     *temperature = ((17572 * raw) / 65536) - 4685;
 }
+
+uint64_t si70xx_get_serial(const si70xx_t *dev)
```

Comment:
If they are only a wrapper to invoke the static function, why not move the static functions to here?

Reply:
Sure, I was just adding these functions because they were in the header but not implemented in the .c file.